lib/collections/schemas/helpers.js

```diff
@@ -55,3 +56,25 @@ export function schemaIdAutoValue() {
   }
   return this.unset();
 }
+
+/**
+ * shopDefaulCountry
+ * @summary used for schema injection autoValue
+ * @example autoValue: shopDefaultCountry
+ * @return {String} returns country value from default shop
+ */
+export function shopDefaultCountry() {
+  if (this.isSet && Meteor.isServer) {
```

Comment:
I can read through this, but I think it's probably worth adding a little more inline commentary as to what's going on for people who are less versed in how autoValue works.